CONTRIBUTING.md

```diff
@@ -93,6 +93,7 @@ NuGet members may contribute directly to the main remote.
 
 - `$(NuGetClientRoot)\artifacts\VS15` - this folder will contain the Package Manager extension (`NuGet.Tools.vsix`) and NuGet command-line client application (`NuGet.exe`)
 - `$(NuGetClientRoot)\artifacts\nupkgs` - this folder will contain all our projects packages
+- `$(NuGetClientRoot)\artifacts\docs-generated` - Containes generated docs. Created with `msbuild .\build\build.proj /t:GenerateDocs`
```

Comment:
I think the docs should be generated into the docs folder at the repo root. 

Is there a good reason why we'd an intermediate doc?

Reply:
Code-generated artifacts normally are not checked-in in a source control repo. I will add the generated artifact.

Reply:
I don't get what you mean.
The doc that this tool is supposed to generate can be a part of the PR. 
It's not a "code-generated" artifact comparable to dlls etc.

Reply:
Moved the project-list.md to docs\ folder